maggy/tests/test_routes_pr_review.py

```
@@ -0,0 +1,82 @@
+class TestRun:
+    def test_missing_token_400(self, client, monkeypatch):
+        monkeypatch.delenv("GITHUB_TOKEN", raising=False)
+        resp = client.post("/api/pr-review/run",
+                           json={"owner": "o", "repo": "r", "pr_number": 1})
+        assert resp.status_code == 400
+
+    def test_run_serializes_verdict(self, client, monkeypatch):
+        monkeypatch.setenv("GITHUB_TOKEN", "tok")
+
+        async def fake_run(owner, repo, num, **kw):
+            assert kw["token"] == "tok"
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to: "token"

(S105)